api/tests/test_manage_asset_api.py

```diff
@@ -444,3 +444,19 @@ def test_admin_cannot_update_assets_to_no_existing_location(
         response = client.get('{}/{}/'.format(self.manage_asset_urls, self.asset.uuid),
                               HTTP_AUTHORIZATION="Token {}".format(self.token_user))
         self.assertEqual(response.data.get('asset_location'), None)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_only_a_superuser_can_update_an_asset_location(self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {'email': self.user.email}
+        data = {
+            "asset_code": "IC003",
+            "serial_number": "SN003",
+            "asset_location": "Nairobi",
+            "model_number": self.assetmodel.model_number}
+        AndelaCentre.objects.create(
+            centre_name="Nairobi",
+            country="Kenya")
+        res = client.put(
+            '{}/{}/'.format(self.manage_asset_urls, self.asset.uuid), data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
```

Comment:
can we create an normal admin user for this test? (`is_staff=True` and is `is_superuser=False`)

Reply:
This has been added.